ftnq.c

```diff
@@ -1130,3 +1130,17 @@ void good_try (FTN_ADDR *fa, char *comment, BINKD_CONFIG *config)
   ++nok;
   write_try (fa, &nok, &nbad, comment, config);
 }
+
+void remove_try (FTN_ADDR *fa, BINKD_CONFIG *config)
+{
+  if (config->tries == 0) return;
+  char buf[MAXPATHLEN + 1];
```

Comment:
Please declare all local variables at the begin of block, after `{`.

Declaration at the random place is allowed by some compilers such as gcc (by default) but it violates C standard.

Reply:
fixed